pptx_blueprint/__init__.py

```diff
@@ -52,6 +57,13 @@ def save(self, filename: __Pathlike):
 
         Args:
             filename (path-like): file name or path
+
+        Raises:
+            TemplateOvewriteError
         """
-        # TODO: make sure that the user does not override the self._template_path
-        pass
\ No newline at end of file
+        if pathlib.Path(filename).absolute() == pathlib.Path(self._template_path).absolute():
+            raise TemplateOvewriteError(
+                f'The specified save path ({filename}) is equal to the path of the template file.' \
+                'The template should not be overwritten.'
```

Comment:
You don't really need the backslash here. I think, it would look better to do it like this:

```python
f'The specified save path ({filename}) is equal to the path of the template file. '
f'The template should not be overwritten.'
```